core/src/main/scala/org/apache/gearpump/util/LogUtil.scala

```diff
@@ -67,6 +67,11 @@ object LogUtil {
     }
   }
 
+  /** getLogger(Class <?>) implementation for Java */
```

Comment:
`<?>` happens to fail scala documentation verification. It's ok to leave it out.

Reply:
I will fix it.